bgp/ocgobgp.go

```diff
@@ -112,6 +144,45 @@ func defaultPolicyToRouteDisp(gobgpdefaultpolicy bgpconfig.DefaultPolicyType) bg
 		return bgpconfig.ROUTE_DISPOSITION_REJECT_ROUTE
 	case bgpconfig.DEFAULT_POLICY_TYPE_ACCEPT_ROUTE:
 		return bgpconfig.ROUTE_DISPOSITION_ACCEPT_ROUTE
+	default:
+		return bgpconfig.ROUTE_DISPOSITION_REJECT_ROUTE
+	}
+}
+
+// convertCommunity converts any community union type to its string representation to be used in GoBGP.
+func convertCommunity(community any) string {
```

Comment:
might be good to start writing some unit tests for these

Reply:
Thanks for the comment, once I have the basic features implemented and the conversion code stable I will add tests. Although since I'm still in the prototyping stage, I'm planning to add them at the higher level (OC -> GoBGP config conversion) rather than at these lower levels where the conversion logic is very straightforward.